### DISCUSSION TITLE
Question on the use of the sleep command

### DISCUSSION THREAD

**Discussion by cmclinton** [jdholtz/auto-southwest-check-in#106] [General]
I started auto-check-in yesterday and it successfully found my flights. I know that (based on the logs) under the hood the program sets up a python sleep command for 'x' of seconds based on when it needs to check the person in. However, my laptop inadvertently went into sleep mode. Would that pause the python sleep command so that when the laptop comes out of sleep mode, the program is off time-wise, i.e. it tries to check-in too late?

**Comment by jdholtz** [comment]
Yes, Python processes are paused while your computer is in sleep. Therefore when it resumes, the time will be off and it will check in too late.

I recommend to run this script on a server, if possible, to mitigate these types of issues. Additionally, it is meant to run in the background without any human intervention once set up, so it doesn't have safeties in-place for situations like this.
